src/InsertOneResult.php

```diff
@@ -35,8 +35,7 @@ class InsertOneResult
     private $isAcknowledged;
 
     /**
-     * @param WriteResult $writeResult
-     * @param mixed       $insertedId
+     * @param mixed $insertedId
```

Comment:
Feel free to remove this if you like.

Reply:
Removed this and a number of other superfluous type annotations.